.github/workflows/integration_models.yml

```diff
@@ -8,23 +8,23 @@ on:
       - release-*
       - feature-*
     paths-ignore:
-      - '**/*.md'
-      - 'docs/**'
-      - 'README.md'
-      - 'Makefile'
-      - 'CONTRIBUTING.md'
-      - 'SECURITY.md'
-      - 'LICENSE'
-      - '.github/**'
-      - 'version.txt'
-      - '.schema/**'
-      - '.vscode/**'
-      - 'deploy/**'
-      - 'install/**'
-      - 'media/**'
-      - 'monitoring/**'
-      - 'acknowledgements.md'
-      - 'Dockerfile*'
+      - "**/*.md"
```

Comment:
I think we should continue using `'` for quotes

Reply:
@Jeadie please fix your prettier to conform to the codebase so we are not changing back and forth.